front-end/src/components/molecules/side-bar/side-bar.tsx

```diff
@@ -1,5 +1,9 @@
+"use client"
+
 import { LayoutGrid, LogOut, Plus } from "lucide-react";
 import Link from "next/link";
+import { useRouter } from "next/navigation";
+import { toast } from "react-toastify";
```

Comment:
I hope you've not installed this `react-toastify` library??
If so, it's best you uninstall it and use the one in the code base.
we're currently using `sonner` for toast.

remember to use the search feature in `VSCode` to quickly check how it's implemented in other pages